interfaces/builtin/ppp.go

```diff
@@ -77,6 +74,10 @@ func (iface *PppInterface) ConnectedSlotSnippet(plug *interfaces.Plug, slot *int
 	return nil, nil
 }
 
+func (iface *PppInterface) KModConnectedPlug(spec *kmod.Specification, plug *interfaces.Plug, slot *interfaces.Slot) error {
```

Comment:
Nice, looking forward to not having to copy-paste empty functions when this work is over :-)